robocop/reports.py

```diff
@@ -223,8 +226,79 @@ def __init__(self):
         self.name = "version"
         self.description = "Returns Robocop version"
 
-    def add_message(self, *args):
-        pass
-
     def get_report(self) -> str:
         return f"\nReport generated by Robocop version: {__version__}"
+
+
+class TimestampReport(Report):
```

Comment:
I have noticed few (really) small issues in docs here (I will point them out in comments) so I have generated docs locally so it's easier to see them:
![image](https://user-images.githubusercontent.com/8532066/179477391-afab131b-7f2b-44f1-825a-911bea6c8bf6.png)

You can also generate docs by running ``tox -e docs`` in the root of the repository. They will be generated in docs/_build/index.html .

Reply:
Thanks. That helped a lot in making docs :smile: